lib/swarm/tracker/tracker.ex

```diff
@@ -698,11 +725,9 @@ defmodule Swarm.Tracker do
                   debug "#{inspect name} has requested to be restarted"
                   {:ok, new_state} = remove_registration(obj, %{state | clock: lclock})
                   send(pid, {:swarm, :die})
-                  case handle_call({:track, name, m, f, a}, nil, %{state | clock: lclock}) do
```

Comment:
@bitwalker I was unsure whether the `state` should be the `new_state` returned from the  `remove_registration/2` function call two lines above? Does it make a difference if the clock is incremented only once for the two operations (remove, then add) rather than twice?